### PR TITLE
Feat/rbac users renamed to principals

### PR DESCRIPTION
**Pull request by jvorcak**

### CODE REVIEW

File: frontend/src/components/pages/acls/Acl.List.tsx [hunk 1]
```diff
@@ -163,19 +163,27 @@ const UsersTab = observer(() => {
                 <Button variant="outline"
                     isDisabled={!Features.createUser}
                     onClick={() => appGlobal.history.push('/security/users/create')}>
-                    Create user
+                    Create Principal
```
Comment (rikimaru0345) [inline frontend/src/components/pages/acls/Acl.List.tsx:166]: this should still be "create user" a prinipal is something abstract.
```suggestion
                    Create User
```

**Review by rikimaru0345 — CHANGES_REQUESTED** [review]

File: frontend/src/components/pages/acls/Acl.List.tsx (hunk 1 above)
Comment (jvorcak) [reply frontend/src/components/pages/acls/Acl.List.tsx:166]: Screenshot 2024-04-24 at 14 24 39

What confuses me how it's called in an empty action in a table.

**Review by rikimaru0345 — APPROVED** [review]

File: frontend/src/components/pages/acls/Acl.List.tsx (hunk 1 above)
Comment (ivpanda) [reply frontend/src/components/pages/acls/Acl.List.tsx:166]: You're right— in the empty table it should be Create User. That's what is being created: a redpanda SASL/SCRAM user.

Comment (jvorcak) [reply frontend/src/components/pages/acls/Acl.List.tsx:166]: @ivpanda @rikimaru0345 so I've changed the buttons to be "Create user" https://github.com/redpanda-data/console/commit/579667c2d7b525ca75934cc88f6fd06d84ae99f1

**Pull request merged**